### PR TITLE
Add github action for continuous builds

### PR DESCRIPTION
**Pull request by bbrto21**
* Build examples of changed packages when create pull request

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Comment by bwikbs** [comment]
OMG ! :scream: 
Now, I know what you're doing! 
I'll test it and let you know! Thank you! I love it! :heart:

File: .github/workflows/build_changed_packages.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            fi
+          done
+          if [[ ${#CHANGED_PACKAGES[@]} -eq 0 ]]; then
+              echo "ENABLE=false" >> "$GITHUB_ENV"
+          else
+              echo "ENABLE=true" >> "$GITHUB_ENV"
+              echo "CHANGED_PACKAGES=${CHANGED_PACKAGES[@]}" >> "$GITHUB_ENV"
+          fi
+      - name: Install prerequisite packages
+        if: contains(env.ENABLE, true)
```
Comment (swift-kim) [inline .github/workflows/build_changed_packages.yml:44]: How about
```suggestion
        if: env.ENABLE == 'true'
```
or just
```suggestion
        if: env.CHANGED_PACKAGES != ''
```

**Review by swift-kim — COMMENTED** [review]
It is quite unfortunate that we have to split the workflows (jobs) into two only because GitHub Actions doesn't support force exiting a job earlier.

How about merging the workflows into one (`build.yml`) and splitting into two jobs like:

```yaml
name: Build

on:
  push:
  pull_request:

jobs:
  push:
    if: github.event_name == 'push'
    ...
  pr:
    if: github.event_name == 'pull_request'
    ...
```

File: .github/workflows/build_all_packages.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install prerequisite packages
+        run: |
+          sudo apt-get install \
```
Comment (swift-kim) [inline .github/workflows/build_all_packages.yml:15]: How about

```suggestion
          sudo apt install -y \
```

just for consistency with the line

```
sudo apt install -y pciutils zip libncurses5 python libpython2.7
```

File: .github/workflows/build_changed_packages.yml (hunk 1 above)
Comment (bbrto21) [reply .github/workflows/build_changed_packages.yml]: Thanks~ I choose 2nd

File: .github/workflows/build_changed_packages.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
+name: Build changed packages
+
+on:
+  pull_request:
```
Comment (swift-kim) [inline .github/workflows/build_changed_packages.yml:4]: Not building on `push`?
```suggestion
  push:
  pull_request:
```

Comment (bbrto21) [reply .github/workflows/build_changed_packages.yml]: Thanks! That's the part I missed.

**Review by swift-kim — APPROVED** [review]

**Review by bwikbs — APPROVED** [review]

**Pull request merged**